internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strings"
 
 	admin20240530 "go.mongodb.org/atlas-sdk/v20240530005/admin"
+	admin20240805 "go.mongodb.org/atlas-sdk/v20240805005/admin"
 	"go.mongodb.org/atlas-sdk/v20241023001/admin"
```

Comment:
are we still using something in adv_cluster from latest SDK?

Reply:
connV2 (lastest SDK) is being used for GET cluster operations, as well as advanced cluster API calls. These have no risk of unintentional side effects.

Reply:
good, thanks.